builds/android/app/src/main/jni/src/Android.mk

```diff
@@ -33,21 +19,21 @@ LOCAL_SRC_FILES := SDL_android_main.c \
 
 LOCAL_SHARED_LIBRARIES := SDL2
 
-LOCAL_STATIC_LIBRARIES := cpufeatures
+LOCAL_STATIC_LIBRARIES := \
+		vorbisfile vorbis ogg WildMidi \
+		xmp-lite mpg123 speexdsp sndfile \
+		freetype pixman-1 png opusfile opus \
+		lcf expat icui18n icuuc icudata \
+		cpufeatures
```

Comment:
I was not sure here if it still uses our patched lib, if we move cpufeatures.

Reply:
Me neither, the problem with cpufeatures was only on some 4.0.3 devices. So we can just wait what happens.